src/main/java/com/test/sb/service/RoleService.java

```diff
@@ -0,0 +1,23 @@
+package com.test.sb.service;
+
+import com.test.sb.model.Role;
+import com.test.sb.repository.RoleRepository;
+import java.util.NoSuchElementException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RoleService {
```

Comment:
same, this should be the implementation of the interface
```suggestion
public class RoleServiceImpl implements RoleService {
```